### PR TITLE
Fixed issues.

### PR DESCRIPTION
**Pull request by cristijigau**
Fixed all mentioned issues.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -57,49 +60,38 @@ const searchRequest = search => {
...
-    if (this.readyState == 4 && this.status == 200) {
-      responseContent = httpRequest.responseText;
-      parsed = JSON.parse(responseContent);
+  httpRequest = new Request(fullURL, { method: 'GET' });
+
+  fetch(httpRequest)
+    .then(function (response) {
+      return response.json();
+    })
+    .then(function (data) {
```
Comment (peBetrator) [inline js/index.js]: Pls use arrow functions

Comment (cristijigau) [reply js/index.js]: Sure, don't know why I've forgotten. I just intended to test this piece of code with regular functions and then switch to arrow functions but I forgot about them :D

**Comment by peBetrator** [comment]
Well done overall 👍

File: js/index.js (hunk 1 above)
Comment (peBetrator) [reply js/index.js]: Yeah, it happens... still not fully believe in ()  => {} 😃

**Review by peBetrator — APPROVED** [review]

File: js/index.js (hunk 1 above)
Comment (cristijigau) [reply js/index.js]: That's true, can't forget about my old friend "function()" yet. 😃

**Pull request merged**